src/main/java/org/dataone/cn/indexer/object/ObjectManager.java

```diff
@@ -73,6 +48,14 @@ public class ObjectManager {
     private static MultipartD1Node d1Node = null;
     private static Session session = null;
 
+    static {
+        try {
+            manager = new ObjectManager();
+        } catch (ServiceFailure | IOException e) {
```

Comment:
If an exception is thrown, it's critical and un-recoverable. I think we should add the exception `e` to the logging statement, too, so it logs the stacktrace. What do you think? should we even do a System.exit(), [like we do here](https://github.com/DataONEorg/dataone-indexer/blob/ea064aacb72a764e7d2720e13177c568566fcdba/src/main/java/org/dataone/cn/indexer/IndexWorker.java#L107)?